### PR TITLE
Updatereadme

### PR DESCRIPTION
**Pull request by bahadasx**
Added instructions for running process to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+Once all of the installation instructions have been followed you can start the Tika service by running the following command:
+```
+java -jar tika-server-1.7.jar --port 9998
+```
+
+##Downloading and Processing files
+In order to download zip files from the USPTO bulk data site (https://bulkdata.uspto.gov/data2/patent/trial/appeal/board/)
+run the retrieval script:
+```
+./retrieval_files.sh
```
Comment (arowla) [inline README.md]: Is this the right file name?

Comment (bahadasx) [inline README.md]: Nope!  Just fixed it and pushed the update.

**Comment by arowla** [comment]
Hey @bahadasx, this PR now contains a lot more than README updates-- was that intentional?

**Comment by bahadasx** [comment]
@arowla All I see is an addition to the .gitignore file when I look in files changed.  Are you seeing anything additional?

**Comment by arowla** [comment]
That is weird. I was seeing a pubs or grants retrieving shell script. Not now. Merging!

**Pull request merged**